src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/QPack/QPackDecoder.cs

```diff
@@ -710,15 +710,15 @@ private void OnIndexedHeaderName(int index)
             _state = State.HeaderValueLength;
         }
 
-        private void OnIndexedHeaderNamePostBase(int index)
+        private static void OnIndexedHeaderNamePostBase(int index)
```

Comment:
These won't be static once they're actually implemented.

Reply:
I think that's fine, one you implemented them you remove static